step.yml

```diff
@@ -261,7 +272,7 @@ inputs:
       category: Debug
       title: "Run xcodebuild test only"
       description: |-
-        If `single_build` is set to false, the step runs `xcodebuild OPTIONS build OPTIONS` before the test 
+        If `single_build` is set to false, the step runs `xcodebuild OPTIONS build OPTIONS` before the test
         to generate the project derived data. After that comes `xcodebuild OPTIONS build test OPTIONS`. This command's log is presented in the step's log.
```

Comment:
After that comes `xcodebuild OPTIONS build test OPTIONS`. >> This is followed by `xcodebuild OPTIONS build test OPTIONS`.

Reply:
step > Step